apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
+          // text: does `question` contain `text`?
+          // answer: does `question` contain `answer`? // key: `answer` or `userAnswer`? (Yes)
+          // message: why do we need `message` again?
+          // questionHash: why do we need `questionHash` again?
+          registryAnswer: getRegistryAnswer(question.id), // key: `registryAnswer` or `registerAnswer`
+        });
+        return acc;
+      }, []);
+    };
+
+    const mutationData = {
+      environment: "test/acc/prod", // Send from client?
+      date: new Date(), // Send from client?
+
+      environmentUrl: window.location.href,
+      topic: topic,
+      checkVersion: "1.0", // getCheckVerion()
```

Comment:
Is this the version of our application or the version of the sttr-file? I think we need both.
Application version (from package.json) is available in process.env.

Reply:
Do we really need both? We just decided today that we only keep the activity/sttr version in the XML files. Not in the JSON. So:
```
// How in the world are we going to get STTR version(s)?
// In topic.sttrFile we know the JSON file, but that's it
// If we really want the STTR version, we can do that 2 ways:
// 1) We go into topics.source.json and go get all activity id's and with those go into the xml files
// 2) (Re)decide to add them into the [topic].json file
```